test/visual_regression/services/visual_testing/visual_testing.ts

```diff
@@ -84,5 +85,10 @@ export async function VisualTestingProvider({ getService }: FtrProviderContext)
         ? snapshot
         : await browser.execute<[], string>(takePercySnapshotWithAgent);
     }
+
+    public async showElementInPercy(selector: string) {
+      const el = await find.byCssSelector(selector);
```

Comment:
Please use test subjects for all selectors instead of css selectors, css selectors should only be used when we don't have other options or control over the specific piece of UI we're trying to interact with (which shouldn't be the case for visual regression tests).